### DISCUSSION TITLE
List of games

### DISCUSSION THREAD

**Discussion by hakanuzunoglu** [sjdawson/GentlemanDriverPlugin#11] [General]
It would be great if we could get a list of compatible games. or even per property, if we could get which games they work in etc.

**Comment by sjdawson** [comment]
Hi @hakanuzunoglu,

I've tried to make these properties work in all games where possible, at the moment, there's only Tyre Compound that's locked to F1 2020 - mostly as I need to generate the maps for those manually and haven't had the chance.

Is there a specific property/game combination you've tried that doesn't work?

**Reply by hakanuzunoglu** [reply]
I didn't try anything yet to be honest. I just don't know which games are supported so I didn't try by game yet.

**Reply by sjdawson** [reply]
No worries. Best to give it a shot in that regard, see if the property works for the game you're running. If you run into issues, pop back and let me know so I can try to fix it for you. 👍🏻
